dashboard/dashboard-ops-deploy.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Deploy TiDB Dashboard
+category: how-to
+---
+
+# Deploy TiDB Dashboard
+
+The TiDB Dashboard interface is built into the PD component of TiDB v4.0 or higher, and no additional deployment is required. You can just deploy a standard TiDB cluster, and TiDB Dashboard is integrated natively.
+
+See the following documents to learn how to deploy a standard TiDB cluster:
+
++ [Quick Start Guide for the TiDB Database Platform](/quick-start-with-tidb.md#deploy-a-local-test-environment-using-tiup-playground)
++ [Deploy TiDB in Production Environment](/production-deployment-using-tiup.md)
++ [Kubernetes environment deployment](https://pingcap.com/docs/tidb-in-kubernetes/stable/access-dashboard/)
+
+> **Note:**
+>
+> You cannot deploy TiDB Dashboard in a TiDB cluster earlier than v4.0.
+
+## Deployment with multiple PD instances
+
+When multiple PD instances are deployed in the cluster, only one of these instances provides the TiDB Dashboard service.
+
+PD instances, when running for the first time, automatically negotiate with each other and choose one instance to provide the TiDB Dashboard service. After the negotiation is completed, the TiDB Dashboard service will always run on the chosen instance no matter it is restarted or scaled out, unless this instance is manually scaled in. TiDB Dashboard will not run on other PD instances. This negotiation process can be completed automatically without user intervention.
+
+When you access a PD instance that does not provide the TiDB Dashboard service, the browser receives a redirection instruction and automatically guides you to re-access the PD instance that provides the TiDB Dashboard service, so that you can use the service normally. This process is shown in the image below.
```

Comment:
```suggestion
When you access a PD instance that does not serve TiDB Dashboard, the browser will be redirected automatically to guide you access the PD instance that serves the TiDB Dashboard, so that you can access the service normally. This process is illustrated as the image below.
```